### PR TITLE
Add the -no-version option to the toplevel.

### PR DESCRIPTION
**Pull request by shindere**
This option requests the toplevel not to print its version number
at startup.

### CODE REVIEW

File: utils/clflags.ml [hunk 1]
```diff
@@ -67,6 +67,7 @@ and use_threads = ref false             (* -thread *)
 and use_vmthreads = ref false           (* -vmthread *)
 and noassert = ref false                (* -noassert *)
 and verbose = ref false                 (* -verbose *)
+and noversion = ref false                 (* -q *)
```
Comment (damiendoligez) [inline utils/clflags.ml]: You should fix the comment.

Comment (shindere) [inline utils/clflags.ml]: Damien Doligez (2016/05/04 08:00 -0700):

> You should fix the comment.

Oops sorry, fixed!
And sorry for the typo in branch name.

File: manual/manual/cmds/top.etex [hunk 2]
```diff
@@ -246,6 +246,9 @@ Print version string and exit.
 \item["-vnum"]
 Print short version number and exit.
 
+\item["-no-version"]
+Do not print version string at startup.
```
Comment (damiendoligez) [inline manual/manual/cmds/top.etex]: Why is this description different from the one in `ocaml.m`?
IMO they should both say something like "Do not print the version banner at startup."

Comment (shindere) [inline manual/manual/cmds/top.etex]: Fixed and rebased on latest trunk.

**Pull request merged**